great-idea/index.html

```diff
@@ -16,8 +16,77 @@
 
   <body>
 
-    <!-- Copy and paste your HTML from the first UI project here -->
-      
+    <div class="container">
+
+      <header>
+        <nav>
+          <a href="services.html">Services</a>
```

Comment:
you either need to rename your services page to default or put this in your href:
Great%20Idea!%20-%20services.html

Reply:
@NickStrick Thanks, I sent a new commit with the Home and Service pages looking pretty good and linking up together.